node/src/actor_system_factory.rs

```diff
@@ -708,6 +699,11 @@ mod tests {
     use std::sync::{Arc, Mutex};
     use std::thread;
     use std::time::Duration;
+    use lazy_static::lazy_static;
+
+    lazy_static! {
+        static ref CRYPTDE_PAIR: CryptDEPair = CryptDEPair::null();
```

Comment:
Instead of having a single static pair of `CryptDE`s and trying to keep the tests both running in parallel and staying out of each other's way, this is how we do it now.  Each suite of tests has its own static read-only pair of `CryptDENull`s, and any tests in that suite that need `CryptDEReal`s, or to modify `CryptDE`s make their own and keep them local.